sigar_windows.go

```diff
@@ -306,6 +312,45 @@ func GetProcName(pid int) (string, error) {
 
 }
 
+func GetProcCredName(pid int) (string, error) {
+	handle, err := syscall.OpenProcess(syscall.PROCESS_QUERY_INFORMATION, false, uint32(pid))
+
+	defer syscall.CloseHandle(handle)
+	
+	if err != nil {
+		return "", fmt.Errorf("OpenProcess fails with %v", err)
+	}
+	
+	var token syscall.Token
```

Comment:
I think it OK to use `golang.org/x/sys/windows` here and leave the other methods as is.